detox/src/devices/drivers/android/genycloud/GenyCloudDriver.js

```diff
@@ -106,28 +110,34 @@ class GenyCloudDriver extends AndroidDriver {
     }
   }
 
+  static async globalInit() {
+    onSignalExit(() => {
+      const deviceCleanupRegistry = GenyDeviceRegistryFactory.forGlobalShutdown();
```

Comment:
Overall, my only question is why don't you KILL on signal exit? Why only print?

Reply:
Per discussion, the motivation is to leave the devices running if the programmer clicks Ctrl+C with an intention to look at something specific about those currently running devices.